pages/form/Question3.js

```diff
@@ -1,99 +1,46 @@
-      <Layout>
-        <Q3Progress />
-        <Heading textAlign="center">
-          Looks like you are working from home!
-        </Heading>
-        <Text textAlign="center" mb={6}>
-          <br />
-          Move to the next question.
-        </Text>
-        <LinkButton href="/form/Question4">Continue</LinkButton>
-      </Layout>
-    );
-  }
+  const saveAnswers = () => setAnswers((prev) => ({ ...prev, km: km }));
 
   return (
     <Layout>
       <Q3Progress />
-      <Heading>Question 3/5</Heading>
-      <Text textAlign="center" mb={6}>
-        <br />
-        How would you travel to work on a typical week?
-        <br />
-      </Text>
-      <Table variant="striped">
-        <Thead>
-          <Tr>
-            <Th />
-            {travelDays.map((day) => (
-              <Th key={day} textAlign="center">
-                {day}
-              </Th>
-            ))}
-          </Tr>
-        </Thead>
-        <Tbody>
-          {modesOfTransport.map((mode) => (
-            <Tr key={mode}>
-              <Td>{capitalize(mode)}</Td>
-              {travelDays.map((day) => (
-                <Td key={day} textAlign="center">
-                  <Checkbox
-                    value={mode}
-                    onChange={(v) => {
-                      const arr = [...modes];
-                      if (v.target.checked) {
-                        arr[daysOfWeek.indexOf(day)] = v.target.value;
-                        setModes(arr);
-                      }
-                    }}
-                  />
-                </Td>
-              ))}
-            </Tr>
-          ))}
-        </Tbody>
-      </Table>
-      <LinkButton
-        href="/form/Question4"
-        onClick={() =>
-          setAnswers((prev) => ({
-            ...prev,
-            transportModes: modes,
-          }))
-        }
-        disabled={modes.every((m) => m === "didNotTravel")}
-      >
-        Continue
-      </LinkButton>
+      <Heading as="h1" size="md" mt="6">About how many kilometres is your average work commute (one way)?</Heading>
+      <Text mt="4">You can use <Link href="https://maps.google.com.au" color="blue" isExternal={true}>Google Maps</Link> to measure the distance from your home to usual workplace.</Text>
+      <Text mt="4">If you currently work from home, let us know how far it is from your home to your usual workplace.</Text>
+      <FormControl w="50%" alignSelf="start" mt="4">
+        <NumberInput isRequired={true}>
+          <NumberInputField
+            id="km-input"
+            placeholder="Distance in kms"
+            value="25"
+            onChange={(e) => setKm(e.target.value)}
+          />
+        </NumberInput>
+        <FormHelperText id="km-input-helper">*Required</FormHelperText>
+      </FormControl>
+      <Grid templateColumns="repeat(2, 1fr)" gap={4}>
+        <BackButton href="/form/Question2" onClick={saveAnswers} />
+        <ContinueButton href="/form/Question4" onClick={saveAnswers} />
```

Comment:
Hi @ymloh ,

Are you missing a `disabled` parameter in `<ContinueButton />` component?
The 'Continue' button is currently not disabled when the respondent hasn't provided a value in text field

Also, i'm not sure why, but I can't input a value in the text field!!
Can you try out the app with your changes here, and lemme know if you get to same problems?
https://council-emissions-calculator-spike-git-fork-605881-civic-makers.vercel.app/council-emissions-calculator-spike/

Reply:
Oh wow, that's pretty bad, sorry about that! My head must have been someplace else entirely.
I'm on it! ⛑️

Reply:
@hqtan I used a Number Field for Question 3's input, so it won't accept non-numeric input. Is this the issue? Should I change it to string input?

Reply:
oooh gotcha, sorry @ymloh ! I probably didn't notice that! :bow: